tests/functional/frontend/AnswerOptionsFunctionsPluginTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace ls\tests;
+
+use Facebook\WebDriver\WebDriverBy;
+
+/**
+ * @since 2021-11-24
+ * @group plugins
+ */
+class AnswerOptionsFunctionsPluginTest extends TestBaseClassWeb
+{
+
+    /**
+     * @inheritdoc
+     * Activate needed plugins
+     * Import survey in tests/surveys/.
+     */
+    public static function setUpBeforeClass(): void
+    {
+        parent::setUpBeforeClass();
+        self::_activateAndLoadPlugins();
+        $surveyFile = self::$surveysFolder . '/limesurvey_survey_expressionAnswerOptionsTest.lss';
+        self::importSurvey($surveyFile);
+
+    }
+
+    /**
+     * Check if function return good value in loaded survey
+     **/
+    public function testGetAnswerOptionText()
+    {
+        $urlMan = \Yii::app()->urlManager;
+        $urlMan->setBaseUrl('http://' . self::$domain . '/index.php');
+
+        $testToDo = array(
```

Comment:
Tiny details, but personally I prefer the `[]` notation above the older `array()`.

Reply:
Personally : i prefer array( … )

A ruleset for this ?

Reply:
Any reason you prefer the old notation? :)

Have to check for conventions...

Reply:
I'm old too ;) 

but : 
array() : «OK, it's an array»

boolval($toto) : ok i get a boolean value

etc  …

Reply:
https://www.reddit.com/r/PHP/comments/3qekc8/cs_array_or/

Reply:
https://www.reddit.com/r/PHP/comments/3qekc8/comment/cwel0ay/?utm_source=share&utm_medium=web2x&context=3 

But : need to be set in a ruleset.

Reply:
Yep, will add a task for it

Reply:
Minimum : https://manual.limesurvey.org/Code_quality_guide , if it was in manual : i respect :)